packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1867,14 +1870,9 @@ export class _Exporter {
      */
     private _isBabylonCoordinateSystemConvertingNode(node: Node): boolean {
         if (node instanceof TransformNode) {
-            if (node.name !== "__root__") {
-                return false;
-            }
-
             // Transform
-            const matrix = node.getWorldMatrix();
-
-            if (matrix.determinant() === 1) {
+            const matrix = node.getWorldMatrix().multiplyToRef(convertHandednessMatrix, TmpVectors.Matrix[0]);
+            if (!matrix.isIdentity()) {

```

Comment:
So, a regular mesh in the hierarchy that would have `scaling.x = -1` with position=(0,0,0) and quaternion=Identity will also be considered as a coordinate system converting node?

That may be the intent of the PR but I prefer asking!

Reply:
Does it not mean that any rotation scale offset as we fo in the pg from time to time would fail ?

Reply:
> So, a regular mesh in the hierarchy that would have scaling.x = -1 with position=(0,0,0) and quaternion=Identity will also be considered as a coordinate system converting node?

Yes. This only checks root nodes.

> Does it not mean that any rotation scale offset as we fo in the pg from time to time would fail ?

Yes. It's supposed to fail. If the root node doesn't result in identity, then we have to keep it.

Reply:
I'm not sure I understand your answer.

If I have the `__root__` node and below it another node that is a mesh with `scaling.x=1` / `position=(0,0,0)` / `quaternion=Identity`, wouldn't that node be considered a coordinate system conversion node and not serialized (so the mesh is not serialized)?

Reply:
No, we only check root nodes, plus we check to make sure there is no geometry.